scrapers/koji_scraper.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-License-Identifier: GPL-3.0+
+from scrapers.base import BaseScraper
+from purview.models.koji import KojiBuild, KojiTask
+from purview.models.user import User
+from datetime import datetime
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class KojiScraper(BaseScraper):
+
+    def run(self, since=None):
+        log.info('Connected to TEIID, fetching initial load for koji')
+        # Initialize a start date from which all builds must be fetched
+        # Currently hardcoding this to fetch builds from the past two years
+        if since is None:
+            since = datetime(2016, 3, 1)
```

Comment:
Seems like my comment was lost. Can you add the variable `default_since = datetime(2016, 3, 1)` to the `BaseScrapper` class so we can reuse a default date across scrapers.